TodoApp/Tests/TodoApp.Services.Tests/Wrappers/GuidServiceTests.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using NUnit.Framework;
+using TodoApp.Contracts.Wrappers;
+using TodoApp.Services.Wrappers;
+
+namespace TodoApp.Services.Tests.Wrappers
+{
+    [TestFixture]
+    public class GuidServiceTests
+    {
+        private IGuidWrapper _guidWrapper;
+
+        [SetUp]
+        public void SetUp()
+        {
+            _guidWrapper = new GuidWrapper();
+        }
+
+        [Test]
+        public void GenerateGuid_ReturnsDifferentIds()
+        {
+            var id1 = _guidWrapper.GenerateGuid();
+            var id2 = _guidWrapper.GenerateGuid();
+            var id3 = _guidWrapper.GenerateGuid();
+
+            Assert.Multiple(() =>
+            {
+                Assert.That(id1, Is.Not.EqualTo(id2));
+                Assert.That(id1, Is.Not.EqualTo(id3));
+                Assert.That(id2, Is.Not.EqualTo(id3));
+            });
```

Comment:
What if I wanted to be sure that the first `5` items are different? What if I wanted first `20` items to be unique? What about `300`?

There is alternative approach (with multiple flavors, I just picked the one below):
```
const int numberOfIds = 5;

var multipleIds = Enumerable
    .Repeat<Func<Guid>>(_guidWrapper.GenerateGuid, numberOfIds)
    .Select(generateId => generateId())
    .ToArray();

Assert.Multiple(() =>{
    Assert.That(multipleIds, Is.Unique);
    Assert.That(multipleIds, Does.Not.Contain(Guid.Empty));
});
```